spec/helper/PlayerEvent.ts

```diff
@@ -0,0 +1,807 @@
+export enum PlayerEvent {
```

Comment:
Not a big fan of this. Why can't you use the `PlayerEvent` enum from the player package?

Reply:
If we import the enum from `bitmovin-player` we can't mock it off and then the whole player code is executed and all the browser specific variables would have to be mocked as well.

We can't import the enum form the types, as `d.ts` files are not compiled by the `tsc`.

Reply:
Sad but true